src/utils/hedera.ts

```diff
@@ -367,6 +376,26 @@ class Hedera {
     }
   }
 
+  public async getNftInfo(address: string | undefined, account: string | undefined) {
+    if (!address || !account) {
+      return [] as { metadata: string; serial_number: number }[];
+    }
+
+    const addressId = this.hederaId(address);
+    const accountId = this.hederaId(account);
+
+    try {
+      const response = await this.call({
+        url: `/api/v1/tokens/${addressId}/nfts?account.id=${accountId}`,
+        method: 'GET',
+      });
+      return response?.['nfts'] as { metadata: string; serial_number: number }[];
```

Comment:
you need to type at `this.call` soomething like
`const response = await this.call<SomeTypeHere>({` to make it more typesafe.
please check in this file how we are doing it.